File: squad-bsa_Manar-Adel_Python-Part2/squad-bsa_Manar-Adel_BetterCalculator.py
#Better Calculator

def calc(num1,num2,op):
    if op =="+":
        print(num1+num2)
    elif op =="-":
     print(num1-num2)
    elif op =="*":
        print(num1*num2)    
    elif op =="/":
        if num2==0:
            print("INVALID") #DIVIDE BY XERO EXCEPTION
        else:   
            print(num1/num2)    
    else:
        print("INVALID OPERATOR")



num1=float(input("Enter the first number: "))
op=input("Enter the operator: ")
num2=float(input("Enter the second number: "))

calc(num1,num2,op)      

File: squad-bsa_Manar-Adel_Cprogramming-Part2/squad-bsa_Manar-Adel_BetterCalculator.c
#include<stdio.h>

//Better Calculator

void calc(double num1, char op, double num2){

    

    if(op=='+')
        printf("Answer: %f",num1+num2);
    else if(op=='-')
        printf("Answer: %f",num1-num2);
    else if (op=='*')
        printf("Answer: %f",num1*num2);
    else if (op=='/'){
         if(num2==0)
            printf("INVALID"); //divide by zero exception
         else
            printf("Answer: %f",num1/num2); 
}
    else
        printf("INVALID\n");

}


int main(){

    double num1, num2;
    char op;

    printf("Enter the first number: ");
    scanf("%lf",&num1);
    printf("Enter an operator: ");
    scanf(" %c",&op);
    printf("Enter the second number: ");
    scanf("%lf",&num2);

    calc(num1,op,num2);
}